ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge-no-common-path-group.cfg

```diff
@@ -266,6 +266,27 @@ router bgp 65000
       bgp additional-paths receive
       bgp additional-paths send any
       neighbor WAN-OVERLAY-PEERS activate
+   !
+   vrf default
```

Comment:
I'm not sure you would want this config on the default VRF, since it will encapsulate all default routes into EVPN. We have a lot of logic to handle this case for regular DC, to avoid using EVPN for links and loopbacks.